spec/truffle/process/euid_spec.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
+
+require_relative '../../ruby/spec_helper'
+require 'etc'
```

Comment:
Move this inside the `it` itself, because core specs in general should not require stdlibs eagerly.
Also this should be under `spec/ruby/core/process`, not `spec/truffle`

Reply:
Thank you, I should have asked where the suite should go.